src/components/group/readme.md

```diff
@@ -0,0 +1,49 @@
+# gx-group
+
+Web Component where you can group related elements of a form field
+
+## Children
```

Comment:
We use the Children section to explain if the children should have certain structure. Check <gx-card> for example. In this case, the web component accepts any kind of children, so I would remove the section entirely.